src/settings/Settings.tsx

```diff
@@ -123,22 +153,39 @@ export default function Settings() {
     ]
   };
 
+  const bufferStoragePathPlaceholder = state.bufferStoragePath ? state.bufferStoragePath : "(Default)";
```

Comment:
Can we make this default to the real default path rather than "(Default)"?

```
const bufferStoragePathPlaceholder = state.bufferStoragePath ? state.bufferStoragePath : state.storagePath + "/.temp";
```

Reply:
I actually did that at first, but I don't like having to hard-code the _implied_ default on the settings page because it means it has to be updated more than one place should it ever change.

I guess I could make a helper function like `getDefaultBufferStoragePath()`  - I'd be fine with that.

Reply:
Not totally sure I'm with you here. If you change as above, doesn't that pass down to the settings page?

Reply:
Oh, think I understand - personally OK with it being in two places. If we ever update it and forget to change it here then worst case is that it will look a bit odd in the settings but no functional impact. Perhaps add a comment to both locations to say "also see here" to avoid that?

Reply:
Alternatively what you are suggesting with a `getDefaultBufferStoragePath` func sounds great too.